tests/queries/0_stateless/03144_postgresql_dialect.sql

```diff
@@ -0,0 +1,9 @@
+SET dialect = 'postgresql'
+
+SELECT 1
```

Comment:
All four of these queries work in ClickHouse as well.
It means that the test does not show anything about the PostgreSQL query language.